### PR TITLE
Support dark mode and session for sql, minor bug fixes

### LINKED ISSUE #105: (OUI Next Theme) Query Workbench
_This issue to be transferred to corresponding repository_

I am working on launching new light and dark mode themes provided by [OUI component library](https://oui.opensearch.org/1.2/#/) for a target launch within 2.10. These changes support the vision expressed in [Future Vision for Dashboards](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4298.)

I have identified the following front end related issues that prevent the theme from appearing complete and potential solutions within this feature:

- [ ] When in dark mode, editor appears in light mode

### PR DESCRIPTION
**Pull request by ps48**
### Description
1. Dark mode update
2. Removes custom theme partially for Oui compliance 
3. Session Id enabled
4. Async query poll interval time changed to 2 seconds
 
### Issues Resolved
#105 
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — APPROVED** [review]

File: common/utils/async_query_helpers.ts [hunk 1]
```diff
@@ -4,24 +4,41 @@
...
+};
+
+export const getAsyncSessionId = () => {
+  return sessionStorage.getItem(ASYNC_QUERY_SESSION_ID);
+};
+
+export const getJobId = (query: {}, http: CoreStart['http'], callback) => {
+  let id;
+  http
+    .post(`/api/spark_sql_console`, {
```
Comment (derek-ho) [inline common/utils/async_query_helpers.ts:22]: const?

Comment (ps48) [reply common/utils/async_query_helpers.ts]: updated

File: public/utils/PanelWrapper.tsx [hunk 2]
```diff
@@ -3,13 +3,21 @@
...
+  shouldWrap: boolean;
+  children: React.ReactNode;
+}) {
+  return shouldWrap ? (
+    <div style={{ padding: 25 }}>
       <EuiPanel paddingSize="none">{children}</EuiPanel>
-    </div> : <>{children}</>;
+    </div>
+  ) : (
+    <>{children}</>
```
Comment (mengweieric) [inline public/utils/PanelWrapper.tsx:21]: do we need the same wrapping div here if shouldWrap == false?, just thinking if we need to keep the same styling regardless of shouldWrap flag

**Review by mengweieric — APPROVED** [review]

File: public/utils/PanelWrapper.tsx (hunk 2 above)
Comment (ps48) [reply public/utils/PanelWrapper.tsx:21]: I'll recheck in a bit, this is some legacy code from workbench.

Comment (ps48) [reply public/utils/PanelWrapper.tsx:21]: Checked seems like we are only using this for SQL. I'll keep this as is for now. Will have to remove this piece during refactoring.

**Pull request merged**